sda-commons-server-openapi-example/build.gradle

```diff
@@ -1,5 +1,5 @@
 dependencies {
-  compile project(':sda-commons-server-starter')
+  compile project(':sda-commons-server-starter-swagger')
```

Comment:
I think the new OpenAPI code should reference the new bundle

Reply:
done.